components/SignIn.js

```diff
@@ -163,41 +167,99 @@ export default class SignIn extends React.Component {
-                  minWidth={120}
-                  onChange={({ target }) => {
-                    target.value = target.value.trim();
-                    onEmailChange(target.value);
-                    this.setState({ error: target.validationMessage, showError: false });
-                  }}
-                  onKeyDown={e => {
-                    // See https://github.com/facebook/react/issues/6368
-                    if (e.key === ' ') {
-                      e.preventDefault();
-                    } else if (e.key === 'Enter') {
-                      onEmailChange(e.target.value);
-                      this.setState({ error: e.target.validationMessage, showError: true });
-                    }
-                  }}
-                  onBlur={() => this.setState({ showError: true })}
-                  onInvalid={event => {
-                    event.preventDefault();
-                    this.setState({ error: event.target.validationMessage });
-                  }}
-                  placeholder="e.g., yourname@yourhost.com"
-                  required
-                  value={email}
-                  type="email"
-                  width={1}
-                />
+                <StyledInputField
+                  style={{ display: this.props.passwordRequired ? 'none' : 'block' }}
+                  labelFontWeight={600}
+                  labelFontSize="13px"
+                  alignItems="left"
+                  width="100%"
+                  label={<FormattedMessage id="Form.yourEmail" defaultMessage="Your email address" />}
+                  htmlFor="email"
+                  my={2}
+                >
+                  <StyledInput
+                    error={!!error}
+                    fontSize="14px"
+                    id="email"
+                    autoComplete="email"
+                    name="email"
+                    minWidth={120}
+                    onChange={({ target }) => {
+                      target.value = target.value.trim();
+                      onEmailChange(target.value);
+                      this.setState({ error: target.validationMessage, showError: false });
+                    }}
+                    onKeyDown={e => {
+                      // See https://github.com/facebook/react/issues/6368
+                      if (e.key === ' ') {
+                        e.preventDefault();
+                      } else if (e.key === 'Enter') {
+                        onEmailChange(e.target.value);
+                        this.setState({ error: e.target.validationMessage, showError: true });
+                      }
+                    }}
+                    onBlur={() => this.setState({ showError: true })}
+                    onInvalid={event => {
+                      event.preventDefault();
+                      this.setState({ error: event.target.validationMessage });
+                    }}
+                    placeholder="e.g., yourname@yourhost.com"
```

Comment:
I18n version

```suggestion
                    placeholder={intl.formatMessage(
                      { id: 'examples', defaultMessage: 'e.g., {examples}' },
                      { examples: 'yourname@yourhost.com' },
                    )}
```

Reply:
Will not edit this as it's not related to this PR.